artifacts/480/deployment.json

```diff
@@ -0,0 +1,7 @@
+{
+	"gasPrice": 0,
+	"gasLimit": 0,
+	"signerAddress": "",
```

Comment:
i'd still include a valid signer address and a transaction just in case

Reply:
Should I take the transaction details from any particular network? (ETH/OP/???)

Reply:
I think op is OK, or you can rlp encode a transaction yourself with some easter eggs.

Reply:
Updated with OP details 😅

Reply:
I didn't see this, but I think that we shouldn't use an RLP encoded transaction for a different chain. I would just use `0` gas parameters and `0x` empty bytes for the transaction.

Reply:
My concern about empty bytes is that if we were to add validation/tests, then it's hard to distinguish empty bytes for a pre-deploy from empty, missing bytes

Reply:
To me, `0x` (empty) is different than `undefined` (missing). We can also use marker values on the other fields (`gas*`, `signerAddress`) to indicate a pre-deployed contract.